tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/UsingCultureAttribute.cs

```diff
@@ -27,8 +27,8 @@ public override void After(MethodInfo methodUnderTest)
             Thread.CurrentThread.CurrentCulture = this.originalCulture;
             Thread.CurrentThread.CurrentUICulture = this.originalUiCulture;
 #else
-            CultureInfo.CurrentCulture = this.originalCulture;
-            CultureInfo.CurrentUICulture = this.originalUiCulture;
+            CultureInfo.CurrentCulture = this.originalCulture!;
+            CultureInfo.CurrentUICulture = this.originalUiCulture!;

```

Comment:
We don't have warnings here in the FakeItEasy solution... Any idea why?

Reply:
Anyway, maybe a better approach would be to initialize `originalCulture` and `originalUICulture` with the current culture in the constructor?

Reply:
I do not know why there's no warning in FIE.. I was surprised to see it show up. I can take a look over there sometime.

Happy to initialize in the constructor, if you think it's safe. I just didn't know if the `Before`/`After` are run in a different environment than the attribute code. Do we know if the culture will be the same at constructor time as when the test is run?

Reply:
I think it's safe. Anyway, we'll overwrite it in `Before`, so it probably doesn't matter.

Reply:
This is a good point.

Reply:
I pushed a thing, but I don't like it. Hold on.

Reply:
Okay. I like the new thing a little more than the old thing. Carry on.